builder/format.go

```diff
@@ -152,7 +152,7 @@ func (b *Builder) UnstageMixFromBump() error {
 
 // CheckBumpNeeded returns nil if it successfully deduces there is no format
 // bump boundary being crossed.
-func (b *Builder) CheckBumpNeeded() (bool, error) {
+func (b *Builder) CheckBumpNeeded(loop bool) (bool, error) {
```

Comment:
I don't love this approach, as it a) makes the calling sites more confusing, and b) makes _any_ place that uses this function (e.g., the root `PreRun`) have to think about "looping", when "looping" is only a concept that makes sense for `mixer build upstream-format`.

At minimum I'd want this variable changed to `warn` or `silent` or something, or you could make `CheckBumpNeededSilent` do everything this function does except the warning, and then make `CheckBumpNeeded` just call the silent version and then print a warning before returning.

Reply:
Sure I can rename it.